guides/common/modules/proc_registering-hosts.adoc

```diff
@@ -91,17 +91,21 @@ If set to `Yes`, public SSH keys will be installed on the registered host.
 The inherited value is based on the `host_registration_remote_execution` parameter.
 It can be inherited, for example from a host group, an operating system, or an organization.
 When overridden, the selected value will be stored on host parameter level.
-ifdef::katello,satellite[]
+ifdef::katello,orcharhino,satellite[]
 . From the *REX pull mode* list, select whether you want to deploy {Project} remote execution pull client.
 +
 If set to `Yes`, {Project} remote execution pull client is installed on the registered host.
 The inherited value is based on the `host_registration_remote_execution_pull` parameter.
 It can be inherited, for example from a host group, an operating system, or an organization.
 When overridden, the selected value is stored on the host parameter level.
-
-The registered host must have access to a repository containing the `foreman_ygg_worker` package.
-ifndef::satellite[]
-This package is shipped in {Project} client repositories.
++
+The registered host must have access to the {Team} {project-client-name} repository.
++
+ifdef::managing-hosts[]
+For more information about the pull mode, see xref:transport-modes-for-remote-execution_{context}[].
+endif::[]
+ifndef::managing-hosts[]
+For more information about the pull mode, see {ManagingHostsDocURL}transport-modes-for-remote-execution_managing-hosts[Transport Modes for Remote Execution] in _{ManagingHostsDocTitle}_.
 endif::[]
 endif::[]
```

Comment:
I think this step should be enabled for all vendors. Opinions?

Reply:
@adamruzicka @maximiliankolb ^^

Reply:
I'd be inclined to say yes

Reply:
Yes.

Reply:
Thanks!